mypy.ini

```diff
@@ -51,6 +51,6 @@ ignore_missing_imports = True
 
 # Even when excluding a module, import issues can show up due to following import
 # https://github.com/python/mypy/issues/11936#issuecomment-1466764006
-[mypy-setuptools.config._validate_pyproject.*,setuptools._distutils.*]
+[mypy-setuptools.config._validate_pyproject.*,setuptools._vendor.*,setuptools._distutils.*]
```

Comment:
Do we know why the exclusions had to change? Presumably the mypy tests were passing before it was disabled.

Reply:
Whilst the CI setup makes it pass, locally on Windows:
```
setuptools\_vendor\packaging\_manylinux.py:187: error: Cannot find implementation or library stub for module named "_manylinux"  [import-not-found]
setuptools\_vendor\packaging\_manylinux.py:187: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```